tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -925,7 +925,7 @@ SpirvEvalInfo SPIRVEmitter::loadIfGLValue(const Expr *expr,
       } else {
         const bool isMat =
             TypeTranslator::isMxNMatrix(exprType, nullptr, &numRows, &numCols);
-        assert(isMat);
+        (void)isMat;
```

Comment:
i think you meant to add `(void)isMat` after the assert rather than replacing the assert?

Reply:
yup